src/index.html

```diff
@@ -1,16 +1,55 @@
 <!DOCTYPE html>
-<html>
+<html lang="en"></html>
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Balsamiq+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="/css/reset.css">
+    <link rel="stylesheet" type="text/css" href="/css/styles.css">
     <title>Activity Tracker</title>
   </head>
   <body>
-    <h1>Activity Tracker</h1>
+    <nav class="nav-bar">
+    <div class="container-one">
+        <a href=""><i class="burger-menu fas fa-bars"></i></a>
+        <select name="user-drop" id="">
+          <option value="" class="name">User's Name</option>
+        </select>
+      </div>
+      <h1 class="challenge"> Today Challenge: 100000 steps!</h1>
+      <div class="container-two">
```

Comment:
Good thinking on addding these two divs so our flex box would work better, maybe look at more semantic names in the future to describe what these sections contain.